### PR TITLE
Documentation for Arcane Archives.

### PR DESCRIPTION
**Pull request by duely**
Here's the docs for Arcane Archives!

### CODE REVIEW

File: docs/Mods/Arcane_Archives/gct.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+### Class
+
+```java
+import mods.arcanearchives.GCT;
+```
+
+#### Methods
+
+```java
+static void addRecipe(
```
Comment (jaredlll08) [inline docs/Mods/Arcane_Archives/gct.md]: I feel like the use of the "static"  may confuse people thinking that this is ZenScript

File: docs/Mods/Arcane_Archives/gct.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  IIngredient[] inputs // the inputs as an array of ingredients
+);
+```
+
+
+---
+
+
+```java
+static void removeRecipe(
```
Comment (jaredlll08) [inline docs/Mods/Arcane_Archives/gct.md]: I feel like the use of the "static" may confuse people thinking that this is ZenScript

File: docs/Mods/Arcane_Archives/gct.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  IItemStack output // the output itemstack to be removed (quantity must match)
+);
+```
+
+
+---
+
+
+```java
+static void replaceRecipe(
```
Comment (jaredlll08) [inline docs/Mods/Arcane_Archives/gct.md]: I feel like the use of the "static" may confuse people thinking that this is ZenScript

**Comment by duely** [comment]
Fair comment, I'll adjust the script to compensate.

**Comment by duely** [comment]
Static has been removed.

**Pull request merged**